scripts/zones/Ghelsba_Outpost/mobs/Kalamainu.lua

```diff
@@ -5,6 +5,24 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.SLEEPRES, 100)
+    mob:setMod(xi.mod.LULLABYRES, 100)
+    mob:setMobMod(xi.mobMod.NO_MOVE, 1) -- lock from moving
+end
+
+entity.onMobEngaged = function(mob, target)
+    mob:setMobMod(xi.mobMod.NO_MOVE, 0) -- unlock from moving
+    GetMobByID(mob:getID() + 1):updateEnmity(target) -- link Kilioa
```

Comment:
its actually possible to just set them to "superlink" via mobmods, which is what bcnms do auto in the core - the reason it doesn't happen in ghelsba is it can't be flagged as a bcnm zone type (it's a special snowflake in that non bcnm thinsg happen in the same zone) 

https://github.com/LandSandBoat/server/blob/ff257cf51c312c5b186e203410318b066ccf3f1c/src/map/zone_entities.cpp#L252

https://github.com/LandSandBoat/server/blob/5f6b9bc26e7c0ee9d7f2fc3b306829daf2d03b16/src/map/utils/mobutils.cpp#L854

long term, I think we need to move away from relying on the zone type for that in the first place.